tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh

```diff
@@ -37,8 +37,8 @@
 # should fail.
 #
 # STRATEGY:
-#	1. Create a file, and set an with an xattr
-#       2. Set the octal file permissions to 000 on the file.
+#	1. Create a file with an xattr
+#       2. Set the file permissions to 000 on the file
```

Comment:
While you're fixing up this commit, please align this properly.

Reply:
Done!